app/models/open_weather_map.rb

```diff
@@ -0,0 +1,45 @@
+COOLDOWN_IN_MINUTES = 10
+OPEN_WEATHER_MAP_PATH = 'http://api.openweathermap.org/data/2.5/forecast/daily?'
```

Comment:
I think I forgot to mention this in the past, but when you're setting the value of a variable like this in ruby that should change you want to add a .freeze to the end of it